tests/migrations/test_commands.py

```diff
@@ -616,6 +617,20 @@ def test_sqlmigrate_for_non_atomic_migration(self):
             self.assertNotIn(connection.ops.start_transaction_sql().lower(), queries)
         self.assertNotIn(connection.ops.end_transaction_sql().lower(), queries)
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_simple_atomic_migration"})
+    def test_sqlmigrate_for_non_transcational_databases(self):
+        """
+        Transaction wrappers aren't shown for databases not supporting transactional DDL.
+        """
+        out = io.StringIO()
+        connection.can_rollback_ddl = False
```

Comment:
This must use `mock.patch` else it will leak between tests as the connection. Also it should be `connection.features.can_rollback_ddl` and not `connection.can_rollback_ddl`.

Reply:
I dont have much idea about how to use `mock.patch` can please link the appropriate resource